src/socketify/wsgi.py

```diff
@@ -255,24 +278,53 @@ def on_data(data_response, response):
             )
             try:
                 for data in app_iter:
-                    if data and not headers_written:
-                        write_headers(headers_set)
+                    if data:
+                        if not headers_written:
+                            write_headers(headers_set)
+                        if is_chunked:
+                            if isinstance(data, bytes):
+                                lib.uws_res_write(ssl, response, data, len(data))
+                            elif isinstance(data, str):
+                                data = data.encode("utf-8")
+                                lib.uws_res_write(ssl, response, data, len(data))
+                        else:
+                            if isinstance(data, str):
+                                data = data.encode("utf-8")
+                            if failed_chunks:
+                                failed_chunks.append(data)
+                            else:
+                                result = lib.uws_res_try_end(
+                                    ssl,
+                                    response,
+                                    data,
+                                    len(data),
+                                    ffi.cast("uintmax_t", content_length),
+                                    0,
+                                )
+                                # this should be very very rare fot HTTP
```

Comment:
```suggestion
                                # this should be very very rare for HTTP
```
I think this was a typo? I hope I'm helping..!